packages/opentelemetry-resource-util/test/index.test.ts

```diff
@@ -337,6 +249,114 @@ describe('mapOtelResourceToMonitoredResource', () => {
+          otelAttributes: {
+            'cloud.platform': platform,
+            'cloud.availability_zone': 'myavailzone',
+            'k8s.cluster.name': 'mycluster',
+            'k8s.namespace.name': 'myns',
+            'k8s.pod.name': 'mypod',
+            'k8s.container.name': 'mycontainer',
+          },
+        },
+
+        {
+          title: `should map to k8s_container with region fallback on ${on}`,
+          otelAttributes: {
+            'cloud.platform': platform,
+            'cloud.region': 'myregion',
+            'k8s.cluster.name': 'mycluster',
+            'k8s.namespace.name': 'myns',
+            'k8s.pod.name': 'mypod',
+            'k8s.container.name': 'mycontainer',
+          },
+        },
+
+        {
+          title: `should map to k8s_pod on ${on}`,
+          otelAttributes: {
+            'cloud.platform': platform,
+            'cloud.availability_zone': 'myavailzone',
+            'k8s.cluster.name': 'mycluster',
+            'k8s.namespace.name': 'myns',
+            'k8s.pod.name': 'mypod',
+          },
+        },
+
+        {
+          title: `should map to k8s_pod with region fallback on ${on}`,
+          otelAttributes: {
+            'cloud.platform': platform,
+            'cloud.region': 'myregion',
+            'k8s.cluster.name': 'mycluster',
+            'k8s.namespace.name': 'myns',
+            'k8s.pod.name': 'mypod',
+          },
+        },
+
+        {
+          title: `should map to k8s_node on ${on}`,
+          otelAttributes: {
+            'cloud.platform': platform,
+            'cloud.availability_zone': 'myavailzone',
+            'k8s.cluster.name': 'mycluster',
+            'k8s.namespace.name': 'myns',
+            'k8s.node.name': 'mynode',
+          },
+        },
+
+        {
+          title: `should map to k8s_node with region fallback on ${on}`,
+          otelAttributes: {
+            'cloud.platform': platform,
+            'cloud.region': 'myregion',
+            'k8s.cluster.name': 'mycluster',
+            'k8s.namespace.name': 'myns',
+            'k8s.node.name': 'mynode',
+          },
+        },
+
+        {
+          title: `should map to k8s_cluster on ${on}`,
+          otelAttributes: {
+            'cloud.platform': platform,
+            'cloud.availability_zone': 'myavailzone',
+            'k8s.cluster.name': 'mycluster',
+            'k8s.namespace.name': 'myns',
+          },
+        },
+
+        {
+          title: `should map to k8s_cluster with region fallback on ${on}`,
+          otelAttributes: {
+            'cloud.platform': platform,
+            'cloud.region': 'myregion',
+            'k8s.cluster.name': 'mycluster',
+            'k8s.namespace.name': 'myns',
+          },
+        },
+      ].forEach(({title, otelAttributes}) => {
+        it(title, () => {
+          const resource = new Resource(otelAttributes);
+          const actual = mapOtelResourceToMonitoredResource(
+            resource,
+            undefined
```

Comment:
```suggestion
```

Reply:
+1, this overload is deprecated.

Reply:
I fix it at this commit. https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/pull/724/commits/8f28d9885719a32709200cc9f5b20f636336728e